rpm/podman.spec

```diff
@@ -32,6 +19,10 @@
 %define copr_build 1
 %endif
 
+%if %{defined rhel}
+%define fips 1
+%endif
```

Comment:
I am not sure if fips is a good condition name, fips can be enabled on fedora as well.

AFAICT the build tag is calling some special code only in the rhel go builds? At least it doe snot compile on fedora with that tag. So I guess just using rhel as condition seems best.
But overall it would likely be great to have some actual docs on what that tag does and why rhel needs it, I guess some specific crypto policy.

Reply:
The question here is if we need `libtrust_openssl` in Fedora and CentOS too - by default?

Reply:
Well it doesn't build on fedora so the answer is we cannot use it on fedora.
And I guess the rhel macro also applies to centos by default?

Reply:
Yes, `centos` and `rhel` macros are defined in CentOS, in RHEL only `rhel`

Reply:
Yes, this buildtag is only for RHEL and CentOS Stream, and both are covered by %{?rhel}.

Reply:
I could rename it to `fips_enabled` instead if `fips` doesn't feel like a good choice.